src/scripts/components/layers/layer-list-item/layer-list-item.styl

```diff
@@ -10,6 +10,13 @@
   &:hover, &:focus
     background-color: rgba($darkGrey4, 0.5)
 
+    .layerTitle
```

Comment:
I think with CSS Modules you could to something like this to avoid the repetition:

Somewhere at the top:
```css
.ellipsis
  display: block
  overflow: hidden
  width: 100%
  text-overflow: ellipsis
  white-space: nowrap
```

And then
```css
.layerTitle
  composes: ellipsis
```

Reply:
Ah truee, wie used something similar in the slider.